go/worker/registration/worker.go

```diff
@@ -531,7 +555,6 @@ func (w *Worker) registerNode(epoch epochtime.EpochTime, hook RegisterNodeHook)
 		w.registrationSigner,
 		w.identity.P2PSigner,
 		w.identity.ConsensusSigner,
-		w.identity.TLSSigner,
```

Comment:
Why has this been removed? It should probably be signed with both signers when `NextCertificate` is non-nil.

Reply:
That was just to make debugging easier before, I've added it back in now :)